pkg/controller/job/job_controller_test.go

```diff
@@ -792,6 +793,10 @@ func checkCompletionIndexesInPods(t *testing.T, control *controller.FakePodContr
 		} else {
 			gotIndexes.Insert(ix)
 		}
+		expectedName := fmt.Sprintf("%s-%d", jobName, ix)
+		if diff := cmp.Equal(expectedName, p.Spec.Hostname); !diff {
+			t.Errorf("Got pod hostname %s, want %s", p.Spec.Hostname, expectedName)
```

Comment:
great, we have to add the same check in integration and E2E test :)

Reply:
done